content/documentation/4-batch-developer-guides/2-batch/2-spring-batch.md

```diff
@@ -8,34 +8,310 @@ description: 'Create a Spring Batch Job'
+    1. _spring.datasource.username_ - the user name to be used for the MySql database. In the sample below it is `root`
+    1. _spring.datasource.password_ - the password to be used for the MySql database. In the sample below it is `password`
+    1. _spring.datasource.driverClassName_ - The driver to use to connect to the MySql database. In the sample below it is `com.mysql.jdbc.Driver'
+    1. _spring.datasource.initialization-mode_ - initializes the database with the BILL_STATEMENTS and BILL_USAGE tables required for this app. In the sample below we state that we `always` want to do this. This will not overwrite the tables if they already exist.
+    1. _spring.batch.initialize-schema_ - initializes the database with the tables required for Spring Batch. In the sample below we state that we `always` want to do this. This will not overwrite the tables if they already exist.
+
+    ```bash
+    $ java -jar target/billrun-1.0.0.BUILD-SNAPSHOT.jar \
+    --spring.datasource.url=jdbc:mysql://localhost:3306/task?useSSL=false \
+    --spring.datasource.username=root \
+    --spring.datasource.password=password \
+    --spring.datasource.driverClassName=com.mysql.jdbc.Driver \
+    --spring.datasource.initialization-mode=always \
+    --spring.batch.initialize-schema=always
+    ```
+
+3.  Log in to the `mysql` container to query the `BILL_STATEMENTS` table.
+    Get the name of the `mysql`pod using`kubectl get pods`, as shown above.
+    Then login:
+
+<!-- Rolling my own to disable erroneous formating -->
+<div class="gatsby-highlight" data-language="bash">
+<pre class="language-bash"><code>$ docker exec -it mysql bash -l
+# mysql -u root -ppassword
+mysql&gt; select * from task.BILL_STATEMENTS;
+</code></pre></div>
+
+The output should look something like:
+
+| id  | first_name | last_name | minutes | data_usage | bill_amount |
+| --- | ---------- | --------- | ------- | ---------- | ----------- |
+| 1   | jane       | doe       | 500     | 1000       | 6.00        |
+| 2   | john       | doe       | 550     | 1500       | 7.00        |
+| 3   | melissa    | smith     | 600     | 1550       | 7.55        |
+| 4   | michael    | smith     | 650     | 1500       | 8.00        |
+| 5   | mary       | jones     | 700     | 1500       | 8.50        |
+
+#### Cleanup
+
+To stop and remove the mysql container running in the docker instance:
+
+1. Execute the following to get the container id.
+
+   ```bash
+   $ docker ps
+   ```
+
+   You should get a result that looks something like this:
+
+   ```bash
+   CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                               NAMES
+   c3ef4c769c33        mysql:5.7.25        "docker-entrypoint.s…"   6 days ago          Up 5 days           0.0.0.0:3306->3306/tcp, 33060/tcp   mysql
+   ```
+
+1. Using the `CONTAINER ID` above remove the container by executing the command below:
```

Comment:
You can use the container name instead of looking up the container id each time.  Also, see my previous comment about not being able to rm a running container.

Reply:
In both docs removed Removed the need for container lookup using container name.   Also added `docker stop`  before `docker rm`